pkg/volume/metrics_du_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"testing"
 
 	utiltesting "k8s.io/kubernetes/pkg/util/testing"
+	volumetesting "k8s.io/kubernetes/pkg/volume/testing"
```

Comment:
here is your dependency cycle, I think.  Maybe you can mark this test as `package volume_test` ?